setup_f5_gke.sh

```diff
@@ -349,11 +349,12 @@ if [ "$cluster_status" != "0" ]; then
       --enable-stackdriver-kubernetes \
       --enable-ip-alias \
       --network "projects/${GCLOUD_PROJECT}/global/networks/default" \
-      --subnetwork "projects/${GCLOUD_PROJECT}/regions/${GCLOUD_ZONE}/subnetworks/default" \
+      --subnetwork "projects/${GCLOUD_PROJECT}/regions/${GCLOUD_REGION}/subnetworks/default" \
       --default-max-pods-per-node "110" \
       --enable-autoscaling --min-nodes "0" --max-nodes "3" \
       --addons HorizontalPodAutoscaling,HttpLoadBalancing \
-      --no-enable-autoupgrade --enable-autorepair
+      --no-enable-autoupgrade --enable-autorepair \
+      --node-locations  "${GCLOUD_REGION}-a","${GCLOUD_REGION}-b","${GCLOUD_REGION}-c"
```

Comment:
This won't work in `us-east` or `europe-west1`, as they are labelled with `b`, `c`, `d` according to: https://cloud.google.com/compute/docs/regions-zones

Reply:
Change the logic for grabbing 3 zones using gcloud commands in case the user doesn't specify the zones.